package.json

```diff
@@ -23,12 +23,13 @@
     "pretest": "npm run lint",
     "test": "npm run test:unit && npm run test:closure && npm run test:dependency && npm run build && npm run clean",
     "posttest": "istanbul report --root coverage text-summary && istanbul check-coverage --lines 95 --statements 95 --branches 95 --functions 95",
-    "screenshot:build": "npm run clean && mkdirp test/screenshot/out/ && webpack --config=test/screenshot/webpack.config.js --progress",
+    "screenshot:build": "npm run screenshot:clean && mkdirp test/screenshot/out/ && webpack --config=test/screenshot/webpack.config.js --progress",
+    "screenshot:clean": "del-cli test/screenshot/out/",
```

Comment:
I suspect `start` should now run `screenshot:clean` instead of `clean`?

Reply:
Done! Good catch, Ken 😁